.vscode/settings.json

```diff
@@ -23,5 +23,6 @@
     "{build,build-internal}/": true,
     "examples/*/build": true
   },
-  "typescript.tsdk": "./node_modules/typescript/lib"
+  "typescript.tsdk": "./node_modules/typescript/lib",
+  "cSpell.words": ["unprefixed"]
```

Comment:
I wouldn't add `cSpell.words`. Feels like an optional thing folks can add themselves. Any context on this?

Reply:
https://github.com/Shopify/polaris-react/pull/4804/commits/1cbf188997fed903e67dafd076ab0ec74b2b67c4